diff -ru linux-4.9.13-clean/net/rfkill/rfkill-gpio.c linux-4.9.13-patch/net/rfkill/rfkill-gpio.c
--- linux-4.9.13-clean/net/rfkill/rfkill-gpio.c	2017-02-26 05:11:18.000000000 -0500
+++ linux-4.9.13-patch/net/rfkill/rfkill-gpio.c	2017-04-24 12:14:28.860838267 -0400
@@ -161,6 +161,7 @@
 
 #ifdef CONFIG_ACPI
 static const struct acpi_device_id rfkill_acpi_match[] = {
+	{ "OBDA8723", RFKILL_TYPE_BLUETOOTH },
 	{ "BCM4752", RFKILL_TYPE_GPS },
 	{ "LNV4752", RFKILL_TYPE_GPS },
 	{ },
